raiden/api/rest.py

```diff
@@ -1845,6 +1853,8 @@ def _deposit(
         result = self.channel_schema.dump(updated_channel_state)
         return api_response(result=result.data)
 
+    @requires_lc_balance
```

Comment:
correct me if im wrong, this function will: 

1. fail if there is no api key
2. fail if balance is 0
3. fail if balance is greater than 0 but insufficient to make the onchain operation

if 3) happens, it will be catched by is_safe_operation. The possible errors are InsufficientFunds (raised by check_why_[operation]_fail and an unexpected one.

is that corect?

Reply:
yes